packages/state-transition/src/epoch/index.ts

```diff
@@ -59,19 +65,31 @@ export function processEpoch(fork: ForkSeq, state: CachedBeaconStateAllForks, ca
     throw new Error("Lodestar does not support this network, parameters don't fit number value inside state.slashings");
   }
 
+  let timer = metrics?.epochTransitionStepTime.startTimer({step: "processJustificationAndFinalization"});
```

Comment:
I don't think it's necessary to use a let here, just declare individual variables. You only need to register labels once